ext/mysqlnd/mysqlnd_ps.c

```diff
@@ -149,8 +148,9 @@ MYSQLND_METHOD(mysqlnd_stmt, get_result)(MYSQLND_STMT * const s)
 	}
 
 	if (stmt->cursor_exists) {
-		/* Silently convert buffered to unbuffered, for now */
-		DBG_RETURN(s->m->use_result(s));
+		/* Prepared statement cursors are not supported as of yet */
+		zend_throw_error(NULL, "%s() cannot be used with cursors",  get_active_function_name());
```

Comment:
We shouldn't be directly throwing inside the mysqlnd layer.

Reply:
What can I do then to prevent the segfault and inform the users that this option is not available?

Reply:
Use `SET_CLIENT_ERROR()` instead?